x-pack/plugins/task_manager/server/kibana_discovery_service/kibana_discovery_service.ts

```diff
@@ -71,18 +73,21 @@ export class KibanaDiscoveryService {
       } catch (e) {
         if (!this.started) {
           this.logger.error(
-            `Kibana Discovery Service couldn't be started and will be retried in ${this.discoveryInterval}ms, error:${e.message}`
+            `Kibana Discovery Service couldn't be started and will be retried in ${retryInterval}ms, error:${e.message}`
           );
         } else {
           this.logger.error(
             `Kibana Discovery Service couldn't update this node's last_seen timestamp. id: ${this.currentNode}, last_seen: ${lastSeen}, error:${e.message}`
           );
         }
+        if (isClusterBlockException(e)) {
```

Comment:
I think this check needs to move up, otherwise the log always says the retryInterval is 10000 ms even if it's actually 60,000

Reply:
Yeah good point, i didn't check that. Fixed it, thanks.